pkg/storage/tsdb/bucketindex/loader.go

```diff
@@ -26,7 +26,7 @@ import (
 const (
 	// readIndexTimeout is the maximum allowed time when reading a single bucket index
 	// from the storage. It's hard-coded to a reasonably high value.
-	readIndexTimeout = 15 * time.Second
+	readIndexTimeout = time.Minute
```

Comment:
[Nit] Can this const now be moved to the one file where it is used, i.e. pkg/storage/tsdb/bucketindex/storage.go? Or should this constant also be used for `updateBlockIndexEntry`?

Reply:
LGTM @dimitarvdimitrov